config/routes.rb

```diff
@@ -3,8 +3,7 @@
 
   root 'venues#index'
 
-  resources :venues, only: [:index, :show]
-
+  resources :venues, only: [:index, :show, :new, :create]
```

Comment:
Cool. Either way is fine for me. I think the pro of "only" was mentioned by Jarvis in a faciliation, that when looking at the code you don't have to hold "what is missing" in your head. Instead, what you see is the routes you have. That may be worth the benefit of slightly longer only statement vs. shorter "Except".